docs/apis-tools/java-client/job-worker.md

```diff
@@ -13,6 +13,18 @@ keywords: ["backpressure", "back-pressure", "back pressure"]
 
 The Java client provides a job worker that handles polling for available jobs. This allows you to focus on writing code to handle the activated jobs.
 
+:::caution REST API limitation
+
+Camunda 8.6.0 introduces job polling via REST API. However, the HTTP client that Camunda Java client 8.6.0
+uses isn't able to maintain a long-living polling connection, i.e. perform long-polling job activation when
+activating jobs larger than the maximum message size, or when additional job activation requests are received
+while the long-polling connection is still open.
+
+If you encounter this issue, consider switching to the Zeebe gRPC protocol for job activation, or use job
+activation via the REST API with long polling disabled.
+
+:::
```

Comment:
```suggestion
:::caution REST API limitation

Job polling via REST API is available in Camunda 8.6.0, but is not suitable for long-lived polling connections (for example, when activating jobs larger than the maximum message size, or when additional job activation requests are received while the long-polling connection is still open).

If you encounter this issue, consider switching to the Zeebe gRPC protocol for job activation, or use job activation via the REST API with long polling disabled.

:::
```

Is this form of the limitation still correct? If explaining the client limitations is necessary, we can leave it as-is - I'd just put the examples in parentheses to make it a bit easier to read.

Reply:
We can also format the examples as a list. I can expand them with the clarifications that @tmetzke suggested.

Reply:
If there is a need for additional examples, I agree the list might be easier to maintain clarity.